load.php

```diff
@@ -1,6 +1,11 @@
 <?php
+/**
+ * Altis Documentation Module.
+ *
+ * @package altis/documentation
+ */
 
-namespace Altis\Documentation; // @codingStandardsIgnoreLine
+namespace Altis\Documentation; // phpcs:ignore
```

Comment:
Should be scoped to just the relevant rule.

Reply:
Wasn't sure we could with this one - it's because the file is called `load.php` but I thought we were going to allow that in the coding standards? It's the same in all the other modules.

Reply:
Yeah, it'll be fixed soonish, but in the short-term should be scoped